llama_stack/templates/meta-reference-gpu/run-with-safety.yaml

```diff
@@ -13,7 +13,7 @@ apis:
 - telemetry
 providers:
   inference:
-  - provider_id: meta-reference-inference
```

Comment:
this seems like a wrong / unintentional change? because now you have _two_ providers with the same ID?

Reply:
yeah reverted, this was for debugging